examples/graphx/floodfill/src/main.c

```diff
@@ -26,7 +26,7 @@ int main(void)
     gfx_Begin();
 
     /* Use the default palettes, this is already set by gfx_Begin */
-    gfx_SetDefaultPalette();
+    gfx_SetDefaultPalette(gfx_8bpp);
```

Comment:
This line should just be removed, it does nothing in this context.

Reply:
done